src/ImageSharp/Formats/Jpeg/Components/Decoder/HuffmanScanDecoder.cs

```diff
@@ -151,6 +151,7 @@ private void ParseBaselineData()
             if (this.componentsCount == this.frame.ComponentCount)
             {
                 this.ParseBaselineDataInterleaved();
+                this.spectralConverter.CommitConversion();
```

Comment:
Before this PR spectral converter did some magic to determine if image was converted based on number off processed rows. But someone wise once said: "explicit is better than implicit" so I've decided to create a separate method to 'seal' conversion. It doesn't look so OOPish but at least it's more readable IMO.